modules/API/pages/built-in-endpoints.adoc

```diff
@@ -739,148 +739,227 @@ $ curl -X GET 'http://localhost:9000/deleted_vertex_check?threadnum=10&verbose=0
+
+==== Sample request
+
+[tabs]
+====
+With secret::
++
+--
+[source.wrap,bash]
+----
+curl -X POST http://localhost:9000/requesttoken \
+     -d '{"secret": "jiokmfqqfu2f95qs6ug85o89rpkneib3", "lifetime": "100000"}'
+----
+--
+With username-password pair::
++
+--
+[source.wrap,console]
+----
+curl --user example_username:example_password \
+     -X POST http://localhost:9000/requesttoken \
+     -d '{"graph": "example_graph"}'
+----
+--
+Response::
++
+--
+[source.wrap,json]
 ----
-curl -X GET "http://localhost:9000/requesttoken?secret=jiokmfqqfu2f95qs6ug85o89rpkneib3&lifetime=1000000"
 {
   "code": "REST-0000",
   "expiration": 1616042814,
   "error": false,
-  "message": "Generate new token successfully.",
+  "message": "Generate new token successfully.\nWarning: Tigergraph Support cannot restore access to secrets/tokens for security reasons. Please save your secret/token and keep it safe and accessible.",
   "token": "tohvf6khjqju8jf0r0l1cohhlm8gi5fq"
 }
-
-curl --user example_username:example_password -X GET "localhost:9000/requesttoken?graph=example_graph"
 ----
+--
+====
 
-==== Parameters:
+[WARNING]
+====
+Keep your tokens accessible in a secure location.
+For security reasons, TigerGraph Support cannot restore access to tokens.
+====
 
-|===
-| Name | Required | Description
+==== Parameters
+None.
 
-| `secret`
-| Yes if `graph` is not supplied
-| User's secret to generate the token.
 
-| `lifetime`
-| No
-| Period of time for which the token is valid measured in seconds. The default value is about 2.6 million (about a month).
+=== Refresh a token
 
-| `graph`
-| Yes if `secret` is not supplied
-| Name of the graph that the token will be valid for.
-|===
+`PUT /requesttoken`
 
-Users can use either `secret` or their username and password to generate a token. If the user does not supply a secret and chooses to use their username and password instead, then the parameter `graph` becomes required.
+A user can refresh a token using either
 
-=== Request a token (`POST`)
+* A secret, which is associated with the token to be refreshed
+* Their username and password in their request header
 
-`POST /requesttoken`
+This endpoint takes a token and refreshes the lifetime of the token.
+The token itself remains unchanged.
 
-You may also use a `POST` request to generate your token. This allows you to avoid exposing your secret in the query string.
+==== Request body
+The endpoint expects a JSON request body in the following format:
 
-==== Sample request:
+[,javascript]
+----
+{
+    "secret": <secret_value>, <1>
+    "graph": <graph_name>, <2>
+    "lifetime": <token_lifetime> <3>
+}
+----
+<1> User's secret to generate the token.
+Required if the request body does not supply `graph`.
+<2> Name of the graph that the token is valid for.
+Required if the request body does not supply `secret`.
+<3> Period of time for which the token is valid measured in seconds. The default value is about 2.6 million (about a month).
```

Comment:
This part doesn't seem to be consistent with the example below. Please update accordingly.